doc/concepts/morphology.rst

```diff
@@ -607,17 +607,7 @@ because it has two children: the dendrites attached to its distal end.
 Supported file formats
 ----------------------
 
-Arbor supports morphologies described using the SWC file format and the NeuroML file format.
-
-SWC
-~~~
-
-See :ref:`formatswc`
-
-NeuroML
-~~~~~~~
-
-See :ref:`formatneuroml`
+See the main menu under the "file formats" caption, or have a look in the ``doc/fileformat`` directory.
```

Comment:
Why not just keep the links? It's easier. They don't need their own headers though.

Reply:
I wanted to avoid having to keep this up to date (because it had already drifted not). Maybe a link to a `doc/fileformat/index` page?